### PR TITLE
Active directory

### PR DESCRIPTION
**Pull request by spence-man**
Added documentation to [Wiki](https://github.com/chapmanu/chap-press/wiki/Active-Directory-Services)

- Configured settings in NADI plugin for Chapman Network Login
- Created a symlink for the `wp-content` folder for source control, `public/content`
- Updates for plugin can be run as normal

### CODE REVIEW

File: config/database.yml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+---
+# This should be filled out and placed in /chap-press/config/database.yml
+# This file is *only* used to generate wp-config.php
+staging:
+  host: 'localhost' 
+  database: 'chappress_dev' 
+  username: 'chappress'
+  password: 'chappress'
```
Comment (tatwell) [inline config/database.yml]: If these need to be updated after being copied to server, values should be changed to more clearly signal that:

```
staging:
  host: 'TBA'
  database: 'chappress_staging'
  username: 'chappress'
  password: 'TBA'
```

File: config/templates/wp-config.php.erb [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+
+
+/******************** ChapPress - MySQL settings **************************/
+
+/** These database values are only to be used for local environment 
+ *  development for the ChapPress project.
+ */
+
+/** The name of the database for WordPress */
+define('DB_NAME',     '<%= database['database'] %>');
```
Comment (tatwell) [inline config/templates/wp-config.php.erb]: I don't think comments are needed for these settings. Var names are clear enough. You can just list them:

```
define('DB_NAME',     '<%= database['database'] %>');
define('DB_USER',     '<%= database['username'] %>');
...
```

**Review by tatwell — APPROVED** [review]
Looks good as far I can tell. I wish there were a way to exclude the plugin files from the PR review.

If there is anything else you think I need to look over, let me know.

**Comment by spence-man** [comment]
@tatwell Sorry I just merged the branch. That commented code is for the logging story. It's commented out because I didn't want it effecting the plugin story. Should I revert the merge and make the edit or just continue on to the logging story?

**Comment by tatwell** [comment]
@spence-man You can go ahead and make it on the `development` branch then move on to logging story.

**Pull request merged**